bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+FP<sub>P</sub> = max(FP<sub>N</sub>, P<sub>GSP</sub>)
+
+This has the following implications:
+
+- _The true price of the collateral backing a new debt position is worth less than if that natural price were used._
+- If the natural price of the collateral continues to fall:
+	- Some old debt positions might have their true collateral ratio fall below unity (1) (i.e. fractional reserve).
+	- It is possible for new debt positions to be created with a true collateral ratio less than unity (1).
+
+|Comparison to Globally Settled Asset|
+|-|
+|New supply cannot be created for a globally-settled asset.|
+
+#### Updating Collateral behind a Debt Position
+
+It is still possible to update the collateral behind a debt position.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Debt positions cannot be updated for a globally-settled asset because they are closed when global settlement is triggered.|
+
+
+#### Destroying/Closing a Debt Positions
+
+It is still possible to close (pay back) a debt position.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Debt positions are automatically closed when global settlement is triggered.|
+
+
+#### <div id="gsp-activated-margin-calls" /> Margin Calls
+
+Debt positions that were margin called prior to the activation of GSP, will remain margin called until their margin call position is matched or until their collateral ratio is raised by either adding collateral or by the price of the collateral increasing.
+
+_New debt positions (i.e. debt positions created after activation of GSP) are less likely to be margin called during GSP than without GSP because the feed price is protecting the least-collateralized debt position's collateral ratio from falling._
+
+|Comparison to Globally Settled Asset|
+|-|
+|Margin calls are not possible because debt positions are closed when global settlement is triggered.|
+
+
+#### <div id="lower-risk-for-new-debt" />New Borrowers Have a Lower Risk
+
+During GSP, a decrease in the feed price is limited by the need to protect _the least-collateralized debt position's CR from falling below MSSR_.  The feed price can decrease only when the least collateralized debt position's CR increases (i.e. which will cause its debt-denominated call price to decrease).  This CR increase can occur either by the debt position's owner (i.e. the borrower) increasing the collateral, or by its margin call order getting partly filled; partial filling of a margin call increases its CR as perceived by the blockchain.
+
+In addition, during significant downtrends of collateral pricing, many debt positions are often being margin called which create large "walls" on the DEX order book which must be filled before P<sub>gs</sub> can decrease significantly.
+
+Therefore, while GSP is activated is a relatively low-risk time for new debt positions to be created because of the difficulty in triggering margin calls on them.
```

Comment:
I think it's best to put a big WARNING to new positions here: if your position's **real** CR is below MCR, when the least-collateralized debt position is gone (be filled or be adjusted to higher CR), your position will enter margin call territory. A wall doesn't actually protect your position because it can get eaten at any time.